spec/browse_cook_spec.rb

```diff
@@ -0,0 +1,60 @@
+require "rails_helper"
+
+feature "browse cook" do
+  scenario "successfully" do
+    cook = Cook.create!(
+      name: "TEST_NAME",
+      cuisine: "TEST_CUISINE",
+      city: "TEST_CITY",
+    )
+
+    visit cook_path(cook)
+
+    expect(page).to have_content "TEST_NAME"
+    expect(page).to have_content "TEST_CUISINE"
+    expect(page).to have_content "TEST_CITY"
+  end
+
+  scenario "and edit successfully" do
+    cook = Cook.create!(
+      name: "TEST_NAME",
+      cuisine: "TEST_CUISINE",
+      city: "TEST_CITY",
+    )
+
+    visit cook_path(cook)
+
+    click_link "Edit"
+
+    fill_in "Name", with: "TEST_DIFFERENT_NAME"
+    click_button "Update Cook"
+
+    expect(page).to have_content "TEST_DIFFERENT_NAME"
+  end
+
+  scenario "and destroy" do
+    cook = Cook.create!(
+      name: "TEST_NAME",
+      cuisine: "TEST_CUISINE",
+      city: "TEST_CITY",
+    )
+
+    visit cook_path(cook)
+
+    click_button "Destroy"
+
+    expect(page).not_to have_content "TEST_DIFFERENT_NAME"
+  end
+
+  scenario "and go back" do
+    cook = Cook.create!(
+      name: "TEST_NAME",
+      cuisine: "TEST_CUISINE",
+      city: "TEST_CITY",
+    )
+
+    visit cook_path(cook)
+    click_link "Back"
+    expect(page).to have_content "Find cooks near you"
```

Comment:
Have you had a look at the `have_title` method?

Reply:
No, but this looks great so I will use this. Thank you!

Reply:
I struggled with the `have_title` method, even after googling and trying a few different ways I was unable to make the test pass. Good to get your advice here. I managed to make the test pass using `have_text`. I'm not sure if that's better than `have_content`. In fact, upon reflection they seem to be very similar indeed.

Reply:
Just to note, you kindly told me at codebar that the `have_title` method requires the argument to be in the head section of the file, not the `h1` tag. I now understand why my test was failing, which is great, thanks!